docs/native/src/scenarios/browse.rst

```diff
@@ -1,3 +1,56 @@
+..  _browse files:
 
-|stub-icon| Browsing, opening, and saving files from |Office iOS|
-=================================================================
+Browsing, opening, and saving files from |Office iOS|
+=====================================================
+
+Once you have finished integration with |Office iOS|, users will be able to browse, open and save files from your storage location directly in the Office apps as well as open their Office files from your storage location. This section describes the user experience for browsing, opening and saving files from |Office iOS|.
+
+Open from |Office iOS|
```

Comment:
All the numbers in this section seem aligned incorrectly. There seems to be a space before each one.